sdk/modelsrepository/Azure.Iot.ModelsRepository/src/ResolverClient.cs

```diff
@@ -100,7 +100,17 @@ public ResolverClient(Uri repositoryUri, ResolverClientOptions options)
             Justification = "<Pending>")]
         public virtual async Task<IDictionary<string, string>> ResolveAsync(string dtmi, CancellationToken cancellationToken = default)
         {
-            return await _repositoryHandler.ProcessAsync(dtmi, cancellationToken).ConfigureAwait(false);
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(ResolverClient)}.{nameof(Resolve)}");
```

Comment:
This is some left over work from when I did the client diagnostics, all APIs need to create their own scope as well.